src/_langs/en/fundamentals/device-access/stickyness/web-app-manifest.markdown

```diff
@@ -136,9 +144,88 @@ Don't worry, if you think users would prefer to view your page as a normal site
 
 <div class="clear"></div>
 
+## Create a splash screen
+
+When you launch your web app from the homescreen a number of things happen behind the
+scenes:
+
+1. Chrome needs to launch,
+2. The renderer that displays the page needs to start up.
+3. Your site is loaded from the network (or cache like ServiceWorker).
+
+While this is happening the screen will be white to the user and will look like it has stalled.
+This becomes especially apparent if you are loading your web page from the network where
+frequently many pages take more than 1 or 2 seconds to get any content visible on the homepage.
+
+To provide a better user experience you can control the color of the screen by adding a
+`background_color` to your manifest and giving it an HTML color value. The color will be used
+by Chrome the instant the web app is launched and will remain on the screen until the web
+apps first render.
+
+Simply set the following in your manifest.
```

Comment:
You could just shorten this to 'For example:'. Otherwise this is redundant with lines 160/161.